### PR TITLE
enhanced cnv wizard field descriptions

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is focused on the adding of enhanced descriptions for 3 fields in the CNV wizard within the Storage and Networking fields.

### CODE REVIEW

**Comment by lmandavi** [comment]
Adding the email that describes enhanced descriptions for the three fields.
[wizard-descriptions_email.pdf](https://github.com/openshift/openshift-docs/files/4340534/wizard-descriptions_email.pdf)

**Comment by lmandavi** [comment]
Requesting Tomas to review the enhanced descriptions.

**Comment by jelkosz** [comment]
lgtm

**Comment by lmandavi** [comment]
Requesting QE Review by Guohua.

**Comment by gouyang** [comment]
LGTM.

**Comment by lmandavi** [comment]
QE review completed and changes were approved. Ready for peer review.

File: modules/cnv-networking-wizard-fields-web.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@
 |Name for the Network Interface Card.
 
 |Model
-|Driver for the Network Interface Card or model for the Network Interface Card.
+|Indicates the model of the Network Interface Card. Supported values are e1000, e1000e, ne2k_pci, pcnet, rtl8139, and virtio.
```
Comment (aburdenthehand) [inline modules/cnv-networking-wizard-fields-web.adoc]: If these are literal values for the Model then each one should be in backticks - 
> `e10001`, e1000e`, etc

File: modules/cnv-storage-wizard-fields-web.adoc [hunk 2]
```diff
@@ -20,9 +20,9 @@
 |Size, in GiB, of the disk.
 
 |Interface
-|Name of the interface.
+|Type of disk device such as virtio, sata, and scsi.
```
Comment (aburdenthehand) [inline modules/cnv-storage-wizard-fields-web.adoc]: VirtIO, SATA, and SCSI

File: modules/cnv-networking-wizard-fields-web.adoc (hunk 1 above)
Comment (aburdenthehand) [reply modules/cnv-networking-wizard-fields-web.adoc]: Unless they're options in a drop-down menu, in which case the markup should be *

**Comment by lmandavi** [comment]
They are selectable options from a drop-down menu. Formatted the values with *. Please review. Commits have been squashed.

**Comment by lmandavi** [comment]
This is the associated defect for this PR:
https://bugzilla.redhat.com/show_bug.cgi?id=1798598

File: modules/cnv-storage-wizard-fields-web.adoc [hunk 3]
```diff
@@ -20,9 +20,9 @@
 |Size, in GiB, of the disk.
 
 |Interface
-|Name of the interface.
+|Type of disk device such as *virtIO*, *SATA*, and *SCSI*.
```
Comment (aburdenthehand) [inline modules/cnv-storage-wizard-fields-web.adoc]: I still think it should be something like "... disk device. Supported interfaces are ..."
Having "such as" implies that there are alternatives not listed.

Comment (lmandavi) [reply modules/cnv-storage-wizard-fields-web.adoc]: Agree. Made the change - please review. Thanks!

**Comment by lmandavi** [comment]
Squashed the commits.

**Comment by aburdenthehand** [comment]
@lmandavi It looks like you missed one - there are still 2 commits.
Let me know when you've squashed these and I can merge it.

**Comment by lmandavi** [comment]
Squashed the commits. Ready for merge. Thanks much!

**Comment by vikram-redhat** [comment]
Hey @lmandavi doesn't look like the squashing was done correctly. It includes content from Serverless as well.

**Comment by lmandavi** [comment]
With guidance from @bergerhoffer, I did a soft reset to remove content that did not belong to this PR (that extraneous content got added during my previous commit). The PR got closed, so I reopened.

In the HTML output, I verified that the correct content was displaying in the networking and storage fields for the cnv wizard:
https://wizard-descriptions--ocpdocs.netlify.com/openshift-enterprise/latest/cnv/cnv_virtual_machines/cnv-create-vms.html#cnv-networking-wizard-fields-web_cnv-create-vms

@vikram-redhat - QE wanted to see the PR without the extraneous content. I can update the defect so that QE is aware that this PR is ready for verification.

**Comment by vikram-redhat** [comment]
@lmandavi  can this be merged or moved along?

**Comment by lmandavi** [comment]
Yes, it can be merged. Please let me know if we need to meet to go over any
questions you have.


Lavanya Mandavilli

Senior Technical Writer, Cloud Platform

Red Hat <https://www.redhat.com/>
<https://www.redhat.com/>

**Comment by vikram-redhat** [comment]
@lmandavi - thanks. Can you please request the aligned team to merge and CP this?

**Pull request merged**